src/service/SentenceFinder.java

```diff
@@ -0,0 +1,19 @@
+package service;
+
+public final class SentenceFinder {
+
+    /*
+        Find the sentence containing the largest number of words in some given text.
+    */
+    public static int getLongestSequenceSize(String s){
+        if(s == null || s.isEmpty()) return 0;
+        int result = 0;
+        String[] sentenceList = s.split("[!?.]+");
+        for(int i = 0; i < sentenceList.length; i++){
```

Comment:
use clean code ideas and split this loop

Reply:
Fixed it